test_app/lib/screens/test_buttons/conference_service_test_buttons.dart

```diff
@@ -226,17 +227,26 @@ class ConferenceServiceTestButtons extends StatelessWidget {
             context, 'Error', error.toString() + stackTrace.toString()));
   }
 
-  void setSpatialPosition(BuildContext context) {
-    _dolbyioCommsSdkFlutterPlugin.conference
-        .getLocalParticipant()
-        .then((participant) =>
-            _dolbyioCommsSdkFlutterPlugin.conference.setSpatialPosition(
+  Future<void> setSpatialValuesDialog(BuildContext context, String spatialValueType) async {
```

Comment:
It will be better to use enum or some constant to define spatial type not string.